src/shared/components/palette/base-field/field-read.component.ts

```diff
@@ -40,7 +42,7 @@ export class FieldReadComponent extends AbstractFieldReadComponent implements On
       // TODO AW 30/12/20 figure out why FixedLists need plainToClassFromExist
       // Added a check to make sure it's NOT already a CaseField and then
       // assigning it back to this.caseField so we don't create separation.
-      if (this.caseField.field_type.type === 'FixedList' && !(this.caseField instanceof CaseField)) {
+      if (FIX_CASEFIELD_FOR.indexOf(this.caseField.field_type.type) > -1 && !(this.caseField instanceof CaseField)) {
```

Comment:
Is the rationale for not using .includes() because of IE support?

Reply:
Yes, I believe so. We've had to do that same thing elsewhere in CCD Case UI Toolkit.